subtests/docker_cli/run_cgroups/run_cgroups.py

```diff
@@ -55,3 +56,42 @@ class memory_no_cgroup(memory_base):
 
 class memory_negative(memory_base):
     pass
+
+
+class cgroup_parent_invalid_name(cgroup_parent_base):
+    """
+    Invalid value for --cgroup-parent option
+    """
+
+    def initialize(self):
+        super(cgroup_parent_invalid_name, self).initialize()
+        self._setup("/foobar",
+                    "",
+                    "Error response from daemon: Cannot start container {cid}:"
+                    " [8] System error: Invalid slice name /foobar")
```

Comment:
Already split. (And, sidenote, I added a `cleanup` to remove stray cgroups). I will push tomorrow, not today, because my brain is shutting down. I'll also look into refactoring all the DockerVersion checks, because with this addition we'll have three and that's too many to duplicate.